ui/yarn.lock

```diff
@@ -2610,10 +2610,10 @@ autolinker@^3.11.0:
   dependencies:
     tslib "^2.3.0"
 
-axios@^1.4.0:
-  version "1.5.1"
-  resolved "https://registry.yarnpkg.com/axios/-/axios-1.5.1.tgz#11fbaa11fc35f431193a9564109c88c1f27b585f"
-  integrity sha512-Q28iYCWzNHjAm+yEAot5QaAMxhMghWLFVf7rRdwhUI+c2jix2DUXjAHXVi+s1ibs3mjPO/cCgbA++3BjD0vP/A==
+axios@1.6.0, axios@^1.4.0:
+  version "1.6.0"
```

Comment:
`1.6.0` is SemVer-compatible with `^1.4.0`. Also if you notice here, this resolution would actually lock us into `1.6.0` as it is an exact version and not a SemVer range.

```suggestion
axios@^1.4.0:
  version "1.6.0"
```